server/routes/homepage/index.ts

```diff
@@ -14,13 +14,14 @@ export default function routes(services: Services): Router {
     const { user } = res.locals
     const eventsData = await services.prisonerProfileService.getPrisonerEventsSummary(user)
     const linksData = await services.linksService.getHomepageLinks(user)
-
+    const establishmentLinksData = getEstablishmentLinksData(user.idToken.establishment.agency_id)
```

Comment:
Rather than passing the entire object through, you could just grab `hideHomepageEventsSummaryAndProfileLinkTile` and pass that into the template

Reply:
Also worth doing something like `user.idToken?.establishment?.agency_id` so if those bits don't exist it'll just return `undefined` rather than an error.